.github/workflows/zitadel.yml

```diff
@@ -5,6 +5,9 @@ on:
       - '**'
     tags-ignore:
       - '**'
+  pull_request:
+    branches:
+      - '**'
```

Comment:
@livio-a I think this will run anytime a push on a brunch happens or a PR is requested

But for the time being I guess it is the easiest way to do this :grin:

Reply:
you're right. we could change branches for push to `main` only